cmd/relayproxy/config/exporter.go

```diff
@@ -59,6 +61,7 @@ const (
 	S3Exporter            ExporterKind = "s3"
 	GoogleStorageExporter ExporterKind = "googleStorage"
 	SQSExporter           ExporterKind = "sqs"
+	KafkaExporter         ExporterKind = "kafka"
```

Comment:
For now your exporter is not working in the relayproxy because `func (r ExporterKind) IsValid() error` is returning an error.

Be sure to add `KafkaExporter` in the list of the valid `ExporterKind`.